flink-core/src/main/java/org/apache/flink/api/common/ArchivedExecutionConfig.java

```diff
@@ -104,4 +105,80 @@ public long getPeriodicMaterializeIntervalMillis() {
     public Map<String, String> getGlobalJobParameters() {
         return globalJobParameters;
     }
+
+    private String getRestartStrategyDescription(Configuration configuration) {
```

Comment:
Can we introduce an internal util class to host these new codes?
And add some tests for it.